src/20-bam/configuration/reader_v2.cc

```diff
@@ -153,8 +155,7 @@ void reader_v2::_load(state::kpis& kpis) {
         }
       }
     } catch (std::exception const& e) {
-      throw exceptions::msg()
-          << "BAM: could not retrieve KPI configuration from DB: " << e.what();
+      throw com::centreon::exceptions::msg_fmt("BAM: could not retrieve KPI configuration from DB: {}", e.what());
```

Comment:
why adding com::centreon:: ???

Reply:
The code does not run without this specification
I think because the code is inside a com::centreon::broker::bam class, the compiler does not recognize the namespace
![image](https://user-images.githubusercontent.com/65180210/85111923-46329980-b215-11ea-8903-4d01017df994.png)